pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -0,0 +1,158 @@
+package verifiable_test
+
+import (
+	"crypto/ed25519"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+)
+
+//nolint:gochecknoglobals
+var privHolderKey = ed25519.PrivateKey{137, 207, 34, 115, 168, 80, 44, 145, 236, 206, 165, 152, 211, 195, 240, 205, 232, 37, 216, 101, 58, 133, 198, 107, 232, 119, 30, 80, 176, 137, 10, 251, 109, 144, 158, 93, 189, 195, 0, 24, 15, 29, 166, 185, 169, 69, 246, 25, 182, 179, 115, 54, 107, 4, 123, 30, 5, 88, 175, 94, 109, 15, 34, 113} //nolint:lll
```

Comment:
can we mention the source of this key as a comment? eg: how it was generated?

Reply:
> // Private key generated by ed25519.GenerateKey(rand.Reader)
added this comment